Sources/Datadog/Core/Feature.swift

```diff
@@ -27,22 +38,22 @@ internal struct FeatureStorage {
     init(
         featureName: String,
         dataFormat: DataFormat,
-        directory: Directory,
+        directories: FeatureDirectories,
         commonDependencies: FeaturesCommonDependencies
     ) {
         let readWriteQueue = DispatchQueue(
             label: "com.datadoghq.ios-sdk-\(featureName)-read-write",
             target: .global(qos: .utility)
         )
-        let orchestrator = FilesOrchestrator(
-            directory: directory,
+        let authorizedFilesOrchestrator = FilesOrchestrator(
+            directory: directories.authorized,
             performance: commonDependencies.performance,
             dateProvider: commonDependencies.dateProvider
         )
 
         self.init(
-            writer: FileWriter(dataFormat: dataFormat, orchestrator: orchestrator, queue: readWriteQueue),
-            reader: FileReader(dataFormat: dataFormat, orchestrator: orchestrator, queue: readWriteQueue)
+            writer: FileWriter(dataFormat: dataFormat, orchestrator: authorizedFilesOrchestrator, queue: readWriteQueue),
```

Comment:
is this the default writer ? Should you use the `autorizedFilesOrchestrator` here ?

Reply:
Yes, it's the default one. This is changed to `ConsentAwareDataWriter` in next commit of this PR. I use `authorized` orchestrator, as I want to make all data authorized by default until we introduce the consent API in public interface at the end of this EPIC 👌 (otherwise all existing integration tests will fail).